package.json

```diff
@@ -36,16 +36,16 @@
   "readme": "README.md",
   "dependencies": {
     "bluebird": "^3.5.5",
-    "cassandra-uuid": "^0.1.0",
     "content-type": "git+https://github.com/wikimedia/content-type#master",
     "entities": "^1.1.2",
     "fast-json-stable-stringify": "^2.0.0",
-    "hyperswitch": "^0.12.3",
+    "hyperswitch": "^0.12.4",
     "jsonwebtoken": "^8.5.1",
     "mediawiki-title": "^0.6.5",
-    "restbase-mod-table-cassandra": "^1.1.4",
+    "restbase-mod-table-cassandra": "^1.2.0",
```

Comment:
Idem

Reply:
1.2.0 pulls in restbase-mod-table-spec 1.2.0 which uses uuid. Version 1.1.4 depends on restbase-mod-table-spec 1.1.0 which would install cassandra-uuid as a dependency. That would be taking a step backwards.

Reply:
Good point. Meta comment: when you reference a different PR, please provide a link to it.

Reply:
Will do going forward.

> Please add some tests for the UUID functionality. Some other comments in-lined.

Added tests